### ISSUE TITLE
Quest: Searching for the Lost Satchel

### ISSUE THREAD

**Issue by underlord-oberon** [plantain-gnocchis/wow-voiceover-tbc-wotlk#78]
Add-on version: AI_VoiceOver_v1.4.5
Data version: AI_VoiceOverData_VanillaExtra_v1.4

Orgrimmar / Ragefire Chasm / NPC: Maur Grimtotem / Quest: Searching for the Lost Satchel
Link: https://www.wowhead.com/wotlk/quest=5722/searching-for-the-lost-satchel

Description: playback ok.

Completion: Despite the voice-over is perfect, the text is a description of actions. The narrator voice is more suitable in this case, instead of the male tauren voice.

**Comment by carevass** [comment]
This should be fixed with new audio in AI_VoiceOverData_VanillaExtra_v1.5.
